### PR TITLE
Warn when `auth.redirectUri` is sent as an option

### PR DESCRIPTION
**Pull request by luisrudge**
### Changes

Make sure devs copying/pasting code from auth0.js get a warning because both options have very similar names

### CODE REVIEW

**Comment by charlesjohnson** [comment]
Instead of exiting 0 with a “WARN: Your script will exit 0 and fail silently,” should it just exit 1 instead?

**Comment by luisrudge** [comment]
@charlesjohnson we can't fail silently now without a breaking change. The warn is the best we can do without breaking existing users

File: src/core/index.js [hunk 1]
```diff
@@ -251,14 +251,19 @@ function extractAuthOptions(options) {
...
     redirectUrl,
     responseMode,
     responseType,
     sso,
     state,
     nonce
-  } =
-    options.auth || {};
+  } = options.auth || {};
+  if (redirectUri) {
```
Comment (lbalmaceda) [inline src/core/index.js]: I'd keep the previous approach that didn't require you to expose a new variable to this function's scope. (rollback that last commit). Maybe keep the comment inside this IF clause @luisrudge

Comment (luisrudge) [reply src/core/index.js]: done

**Pull request merged**